docs/pages/getting-started/linux-server.mdx

```diff
@@ -207,9 +208,10 @@ Visit the provided URL in order to create your Teleport user.
 
   The users that you specify in the `logins` flag (e.g., `root`, `ubuntu` and
   `ec2-user` in our examples) must exist on your Linux machine. Otherwise, you
-  will get authentication errors later in this tutorial.
+  will get authentication errors later in this tutorial. 
```

Comment:
Is the trailing whitespace here intentional?

Reply:
```suggestion
  will get authentication errors later in this tutorial.
```

Reply:
no, I'll go ahead and remove it